### PR TITLE
feat: add donate faq

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Screen Shot 2021-12-04 at 12 35 49 PM

### CODE REVIEW

**Comment by ahmadabdolsaheb** [comment]
@scissorsneedfoodtoo, the FAQ has been added Quincy's article: https://www.freecodecamp.org/news/how-to-donate-to-free-code-camp/

I was wondering if you could audit the pronouns and make it more concise if possible.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
In the news article, Quincy uses the first-person voice, which makes sense because the news layout clearly attributes the writing to him.

On /learn, however, I'm not sure it makes as much sense as it's not directly associated. I'm wondering if we should tweak the copy to use "we" instead?

**Comment by scissorsneedfoodtoo** [comment]
I'm thinking the same thing, @nhcarrigan. It would be better to use "we" throughout, and tweak the tone a bit. I don't think the writing needs to be quite as personal as the original FAQ article.

I'll take a crack at this and make some suggestions.

**Review by scissorsneedfoodtoo — COMMENTED** [review]
Tried to clean up the copy a bit, and found a couple of typos.

Please let me know if you'd like me to take another look.

**Comment by ahmadabdolsaheb** [comment]
This PR is blocked, I will merge it once it receives 2 approvals and the currently deployed A/B test receives sufficient data.

**Comment by ahmadabdolsaheb** [comment]
@nhcarrigan @scissorsneedfoodtoo This should be ready for another round.

**Review by naomi-lgbt — DISMISSED** [review]
This looks good to me. We may want Quincy to sign off on the copy too.

**Comment by ahmadabdolsaheb** [comment]
Thanks for the review. Quincy differed to our judgement on the copy.

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
I do not think you are supposed to put the placeholder for the link text that way?
image

File: client/i18n/locales/english/translations.json [hunk 1]
```diff
@@ -347,7 +346,42 @@
...
+    "how-efficient": "How efficient is freeCodeCamp?",
+    "fcc-budget": "freeCodeCamp's budget is much smaller than most comparable nonprofits. We haven't brought in professional fundraisers. Instead, Quincy does everything himself.",
+    "help-millions": "However, on a budget of only a few hundred thousand dollars per year, we have been able to help millions of people.",
+    "how-one-time":"How can I make a one-time donation?",
+    "one-time": "If you'd prefer to make one-time donations, you can support freeCodeCamp's mission whenever you have cash to spare. You can use <0>this link to donate whatever amount feels right through PayPal<0>.",
+    "wire-transfer": "You can also send money to freeCodeCamp directly through a wire transfer. If you need our wire details, email Quincy at quincy@freecodecamp.org",
+    "does-crypto": "Does freeCodeCamp accept donations in Bitcoin or other cryptocurrencies?",
+    "yes-crypto": "Yes, and we would welcome your cryptocurrency donations. Here are our wallet details:",
+    "can-check": "Can I mail a physical check?",
+    "yes-check": "Yes, we would welcome a check. You can mail it to us at:",
```
Comment (ShaunSHamilton) [inline client/i18n/locales/english/translations.json:365]:
```suggestion
    "can-cheque": "Can I mail a physical cheque?",
    "yes-cheque": "Yes, we would welcome a cheque. You can mail it to us at:",
```

File: client/src/components/Donation/donation-text-components.tsx [hunk 2]
```diff
@@ -53,3 +84,130 @@ export const DonationOptionsAlertText = (): JSX.Element => {
...
+      A: (
+        <>
+          <p>{t('donate.yes-crypto')}</p>
+          <p>{WALLETS}</p>
+        </>
+      )
+    },
+
+    {
+      Q: t('donate.can-check'),
```
Comment (ShaunSHamilton) [inline client/src/components/Donation/donation-text-components.tsx:166]:
```suggestion
      Q: t('donate.can-cheque'),
```

File: client/src/components/Donation/donation-text-components.tsx [hunk 3]
```diff
@@ -53,3 +84,130 @@ export const DonationOptionsAlertText = (): JSX.Element => {
...
+      A: (
+        <>
+          <p>{t('donate.yes-check')}</p>
```
Comment (ShaunSHamilton) [inline client/src/components/Donation/donation-text-components.tsx:169]:
```suggestion
          <p>{t('donate.yes-cheque')}</p>
```

File: client/src/components/Donation/donation-text-components.tsx [hunk 4]
```diff
@@ -53,3 +84,130 @@ export const DonationOptionsAlertText = (): JSX.Element => {
...
+    { Q: t('donate.how-endowment'), A: <p>{t('donate.endowment')}</p> },
+    {
+      Q: t('donate.how-legacy'),
+      A: (
+        <>
+          <p>{t('donate.we-honored')}</p>
+          <p>
+            <blockquote>{t('donate.legacy-gift-message')}</blockquote>
+          </p>
+          <p>{t('donate.thank-wikimedia')}</p>
```
Comment (ShaunSHamilton) [inline client/src/components/Donation/donation-text-components.tsx:193]: Error:
```
Warning: validateDOMNesting(...): <blockquote> cannot appear as a descendant of <p>.
    in blockquote (created by DonationFaqText)
    in p (created by DonationFaqText)
    in div (created by DonationFaqText)
    in div (created by DonationFaqText)
    in DonationFaqText (created by DonatePage)
```

```suggestion
          <p>{t('donate.we-honored')}</p>
          <blockquote cite="https://commons.wikimedia.org/wiki/Main_Page">
              <p>{t('donate.legacy-gift-message')}</p>
          </blockquote>
          <p>{t('donate.thank-wikimedia')}</p>
```

If you do not like the `cite`, just edit - it is not entirely necessary

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Some more changes

File: client/i18n/locales/english/translations.json (hunk 1 above)
Comment (nhcarrigan) [reply client/i18n/locales/english/translations.json:365]: Sorry to say... But our preference for American spellings means this should be `check` 🙂

Comment (ShaunSHamilton) [reply client/i18n/locales/english/translations.json:365]: Wow... I thought that was just plain incorrect

**Comment by ahmadabdolsaheb** [comment]
> I do not think you are supposed to put the placeholder for the link text that way? image

So this is based on the following paragraph from this article: https://www.freecodecamp.org/news/how-to-donate-to-free-code-camp/
Screen Shot 2021-12-20 at 11 47 45 AM

Let me know if you have another preference so I could alter the pr accordingly.

**Comment by ShaunSHamilton** [comment]
> > I do not think you are supposed to put the placeholder for the link text that way? image
> 
> So this is based on the following paragraph from this article: https://www.freecodecamp.org/news/how-to-donate-to-free-code-camp/ Screen Shot 2021-12-20 at 11 47 45 AM
> 
> Let me know if you have another preference so I could alter the pr accordingly.

Sorry for not being clear, but the issue with the above is the final word in the link text (`a` tag)
> ... PayPalplaceholder

File: client/i18n/locales/english/translations.json [hunk 5]
```diff
@@ -347,7 +346,42 @@
...
+    "get-help": "How can I get help with my donations?",
+    "how-transparent": "How transparent is freeCodeCamp.org?",
+    "very-transparent": "Very. We have a Platinum transparency rating from GuideStar.org.",
+    "download-irs": "You can <0>download our IRS Determination Letter here</0>.",
+    "download-990": "You can <0>download our most recent 990 (annual tax report) here</0>.",
+    "how-efficient": "How efficient is freeCodeCamp?",
+    "fcc-budget": "freeCodeCamp's budget is much smaller than most comparable nonprofits. We haven't brought in professional fundraisers. Instead, Quincy does everything himself.",
+    "help-millions": "However, on a budget of only a few hundred thousand dollars per year, we have been able to help millions of people.",
+    "how-one-time":"How can I make a one-time donation?",
+    "one-time": "If you'd prefer to make one-time donations, you can support freeCodeCamp's mission whenever you have cash to spare. You can use <0>this link to donate whatever amount feels right through PayPal<0>.",
```
Comment (ahmadabdolsaheb) [inline client/i18n/locales/english/translations.json:360]:
```suggestion
    "one-time": "If you'd prefer to make one-time donations, you can support freeCodeCamp's mission whenever you have cash to spare. You can use <0>this link to donate whatever amount feels right through PayPal</0>.",
```

File: client/src/components/Donation/donation-text-components.tsx [hunk 6]
```diff
@@ -53,3 +84,130 @@ export const DonationOptionsAlertText = (): JSX.Element => {
...
+        </>
+      )
+    },
+    { Q: t('donate.how-endowment'), A: <p>{t('donate.endowment')}</p> },
+    {
+      Q: t('donate.how-legacy'),
+      A: (
+        <>
+          <p>{t('donate.we-honored')}</p>
+          <blockquote cite="https://commons.wikimedia.org/wiki/Main_Page">
```
Comment (ahmadabdolsaheb) [inline client/src/components/Donation/donation-text-components.tsx:190]:
```suggestion
          <blockquote>
```

**Comment by ahmadabdolsaheb** [comment]
@ShaunSHamilton should be ok now.

**Review by ShaunSHamilton — APPROVED** [review]
Yip. LGTM 👍

**Review by naomi-lgbt — APPROVED** [review]
Happy to get this in.

**Pull request merged**